hack/tools/mdbook/embed/embed.go

```diff
@@ -1,3 +1,4 @@
+//go:build tools
```

Comment:
Is there a way to enforce this going forward?

Reply:
It's enforced by golangci-lint now, and you can use `make lint-fix`